src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/ProjectSystem/DefaultWindowsRazorProjectHost.cs

```diff
@@ -128,6 +128,7 @@ protected override async Task HandleProjectChangeAsync(string sliceDimensions, I
     // Internal for testing
     internal static bool TryGetConfiguration(
         IImmutableDictionary<string, IProjectRuleSnapshot> state,
+        bool? suppressDesignTime,
```

Comment:
idk if it's better to allow null or just have callers always set a value. I opted for the former but reading this method is a little wonky

Reply:
Yea, I think I'm still foggy brained and overthinking things that should just be straightforward 🌫️🧠